modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -246,6 +248,11 @@ public void processOpts() {
             supportingFiles.remove(new SupportingFile("manifest.mustache", projectFolder, "AndroidManifest.xml"));
         } else if ("google-api-client".equals(getLibrary())) {
             additionalProperties.put("jackson", "true");
+
+        } else if (REST_ASSURED.equals(getLibrary()))  {
+            additionalProperties.put("java8", "true");
```

Comment:
Would it be hard to make it compatible with java6. That way it could be used on android.

Reply:
Initially this client is supposed to be comatible with java8 and supporting java6 would mean rewriting most of the functions, which are used by client.